src/azure-cli/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -795,6 +796,15 @@ def build_vmss_resource(cmd, name, naming_prefix, location, tags, overprovision,
     if proximity_placement_group:
         vmss_properties['proximityPlacementGroup'] = {'id': proximity_placement_group}
 
+    if terminate_notification_time is not None:
+        scheduled_events_profile = {
+            'terminateNotificationProfile': {
+                'notBeforeTimeout': terminate_notification_time,
```

Comment:
Something to consider.

In the powershell command line implementation they just allow users to specify the time in minutes, and the powerhsell cmdlet converts to the appropraite time.

Reply:
> 
> 
> Something to consider.
> 
> In the powershell command line implementation they just allow users to specify the time in minutes, and the powerhsell cmdlet converts to the appropraite time.

Good idea! ISO 8601 format is complicated!